lib/Constants.js

```diff
@@ -3,6 +3,7 @@ const Systems = {
   UHT_CONTACTS: 'UHT-Contacts',
   UHT_HOUSING_REGISTER: 'UHT-HousingRegister',
   UHW: 'UHW',
+  UHT: 'UHT',
```

Comment:
I think we should make this "UHT-ActionDiary" because in the future it will give us the ability to filter these items in the UI